frame/assets/src/lib.rs

```diff
@@ -531,6 +553,7 @@ pub mod pallet {
 					sufficients: 0,
 					approvals: 0,
 					is_frozen: false,
+					status: AssetStatus::Live,
```

Comment:
wonder if this will require a transaction to set status for all existing Assets?
what is the current default value?

Reply:
Do you mean if we need a migration for existing assets? 
The default should be live. Wouldn't it be backward compatible, if existing assets don't have the status already?

Reply:
Yeah, this would need a migration

Reply:
@tonyalaribe yes, from what I see, we will need to set Live status to the existing Assets.

I also wonder now the purpose of `is_frozen`, is not it sufficient for destroy case.
The doc says
https://github.com/paritytech/substrate/blob/10fdd9090b69ff6e42a49ef2c203ea080d92020b/frame/assets/src/lib.rs#L992

Reply:
I imagine we don't want to rely on is_frozen, because anyone could just thaw the asset and unfreeze it.  Whereas when you start destroying an asset, there's no going back

Reply:
I don't think `Frozen` needs to be in there, but see how it could. But yeah `thaw_asset_class` shouldn't be allowed if the status is `Destroying`.

Reply:
I just scrolled down a bit this PR and noticed there already is an `AssetStatus::Frozen` and the logic to thaw the asset is correct, sorry for the confusion :)